src/python/WMCore/WMSpec/Steps/Executors/DQMUpload.py

```diff
@@ -37,8 +37,10 @@ class DQMUpload(Executor):
     Execute a DQMUpload Step
 
     """
+    def __init__(self):
+        return
```

Comment:
This is wrong. You're overriding the parent constructor
https://github.com/juztas/WMCore/blob/errors/src/python/WMCore/WMSpec/Steps/Executor.py#L62

and so skipping some classes attributes.

Why did you add it here? I guess it was a pylint complain... if so, next time make sure to make pylint changes only when you are absolutely sure it's correct, otherwise it's a nightmare to review and detect bugs later on.
That's what I do at least, if I'm unsure, I just skip it :)